### PR TITLE
EMCPy plotting tools docstring

### PR DESCRIPTION
**Pull request by kevindougherty-noaa**
## Description

Added docstring to python scripts within the plotting directory

### CODE REVIEW

File: src/eva/plotting/emcpy/diagnostics/vertical_line.py [hunk 1]
```diff
@@ -4,7 +4,32 @@
 import os
 
 
+# --------------------------------------------------------------------------------------------------
+
+
 class VerticalLine():
+    """Creates a vertical line plot based on the provided configuration.
```
Comment (EdwardSafford-NOAA) [inline src/eva/plotting/emcpy/diagnostics/vertical_line.py]: Super minor thing -- in other files the """ is on a separate line.

**Review by EdwardSafford-NOAA — APPROVED** [review]
One super minor issue.  Looks good.

File: src/eva/plotting/emcpy/diagnostics/vertical_line.py (hunk 1 above)
Comment (kevindougherty-noaa) [reply src/eva/plotting/emcpy/diagnostics/vertical_line.py]: Great catch, thank you!

File: src/eva/plotting/emcpy/diagnostics/histogram.py [hunk 2]
```diff
@@ -6,7 +6,37 @@
...
+    """
+    Creates a histogram plot based on the provided configuration and data.
+
+    Args:
+        config (dict): A dictionary containing the configuration for the histogram plot.
+        logger (Logger): An instance of the logger for logging messages.
+        dataobj: An instance of the data object containing input data.
+
+    This class initializes and configures a histogram plot based on the provided configuration and
+    data. The histogram  plot is created using a declarative plotting library from EMCPy
```
Comment (CoryMartin-NOAA) [inline src/eva/plotting/emcpy/diagnostics/histogram.py]: extra space before plot?

Comment (danholdaway) [inline src/eva/plotting/emcpy/diagnostics/histogram.py]:
```suggestion
    data. The histogram plot is created using a declarative plotting library from EMCPy
```

**Review by danholdaway — APPROVED** [review]

**Pull request merged**